### PR TITLE
Moved bin from the source tree to the build tree

### PR DESCRIPTION
**Pull request by oliverdaniell**
Currently the binaries are being created in the source tree, this commit moves them to the build tree.

### CODE REVIEW

**Comment by oliverdaniell** [comment]
Second commit fixes #56

**Comment by oliverdaniell** [comment]
Agreed, I'll make that change

File: easy_profiler_core/CMakeLists.txt [hunk 1]
```diff
@@ -8,7 +8,7 @@ set(EASY_OPTION_PROFILE_SELF           OFF    CACHE BOOL   "Enable self profilin
 set(EASY_OPTION_PROFILE_SELF_BLOCKS_ON OFF    CACHE BOOL   "Storage expand default status (profiler::ON or profiler::OFF)")
 set(EASY_OPTION_LOG                    OFF    CACHE BOOL   "Print errors to stderr")
 set(EASY_OPTION_PREDEFINED_COLORS      ON     CACHE BOOL   "Use predefined set of colors (see profiler_colors.h). If you want to use your own colors palette you can turn this option OFF")
-set(EASY_OPTION_LIB_STATIC             OFF    CACHE BOOL   "Build easy_profiler as static library.")
+set(BUILD_SHARED_LIBS                  OFF    CACHE BOOL   "Build easy_profiler as static library.")
```
Comment (yse) [inline easy_profiler_core/CMakeLists.txt]: I think it will be better if `BUILD_SHARED_LIBS` set by default on `ON`

Comment (oliverdaniell) [inline easy_profiler_core/CMakeLists.txt]: fixed in latest commit

**Pull request merged**